examples/dune

```diff
@@ -1,7 +1,8 @@
 (executables
  (names readme trees sync process deploy irmin_git_store custom_merge push
    custom_graphql)
- (libraries checkseum.c digestif.c irmin irmin-unix))
+ (libraries astring checkseum.c cohttp digestif.c fmt git irmin irmin-git
```

Comment:
do we still need `checkeum.c` and `digestif.c` now that these libraries are using default implemantion? /cc @dinosaure

Reply:
Note that https://github.com/mirage/irmin/pull/873 is still open. We're not yet constraining `digestif` to a version that provides the default implementations.

Reply:
ack